tech-far-hub/src/pages/template-resources-landing.tsx

```diff
@@ -0,0 +1,65 @@
+import SiteLayout from "../components/site-layout";
+import MDXContent from "../components/mdxcontent";
+import { IPageContext } from "../types";
+import * as React from "react";
+import { graphql, PageProps } from "gatsby";
+import { CardGroup, Grid } from "@trussworks/react-uswds";
+import ResouceCard from "../components/resouces-card";
+
+type ResoucesPageProps = PageProps<Queries.ResourcesLandingContextQuery, IPageContext>;
+const ResoucesLandingPage: React.FC<ResoucesPageProps> = ({ data, children, pageContext }: ResoucesPageProps) => {
```

Comment:
Really digging the size of your components! Very clear on single responsibility!